.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Headers are rather denoted as 
```suggestion
### Describe the bug
```

in GH issues.

Reply:
The other headers remained.

Reply:
Ok, removed.

Reply:
I mean they remained as `**header**`. The sections are good just the formatting is off.

Reply:
I don' tget what you want to obtain.
Here is the current preview:

![image](https://user-images.githubusercontent.com/703248/104127686-d7a95a80-5363-11eb-9191-adbc02e07b5c.png)

Reply:
Just to have all the headers the same, as it was before (i.e. when I've written my "Headers are denoted as..." suggestion I was referring to all the headers, not just "Describe the bug"):

```
---
name: Bug report
about: Create a report to help us improve
title: ''
labels: bug
assignees: ''

---

### Describe the bug
<!-- A clear and concise description of what the bug is. -->

### To Reproduce
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '...'
3. Scroll down to '...'
4. See error

### Expected behavior
<!-- A clear and concise description of what you expected to happen. -->

### Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->

```

Sorry if I wasn't clear before. Added the comments too.

Reply:
Headers changed:

![image](https://user-images.githubusercontent.com/703248/104127975-49ce6f00-5365-11eb-8532-665956585a9a.png)

Reply:
Great, thank you!